src/Tribe/Customizer/Global_Elements.php

```diff
@@ -476,49 +478,68 @@ public function setup() {
 	public function register_settings( WP_Customize_Section $section, WP_Customize_Manager $manager ) {
 		$customizer = Tribe__Customizer::instance();
 
+		// Add an heading that is a Control only in name: it does not, actulally, control or save any setting.
+		$manager->add_control( new Heading(
+			$manager,
+			$customizer->get_setting_name( 'accent_color_heading', $section ),
+			[
+				'label'   => esc_html__( 'Set Font Colors', 'the-events-calendar' ),
+				'section' => $section->id,
+			]
+		) );
+
 		$manager->add_setting(
 			$customizer->get_setting_name( 'accent_color', $section ),
-			array(
-				'default'              => '#334AFF',
-				'type'                 => 'option',
-
+			[
+				'default' => '#334AFF',
+				'type'    => 'option',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 17 space(s) between "'type'" and double arrow, but found 4.

<sub>hash: 696141f4c84fd79c598113bc7c6c2c15</sub>